268. Missing Number/solution3.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+Runtime: 26ms
+Memory: 18 MB
+TC : O(logN)
```

Comment:
You are sorting an array using the sort function so the time complexity should be O(N log N).

Reply:
Oops! missed it again, sorry about that